server.js

```diff
@@ -0,0 +1,291 @@
+'use strict';
+
+const http = require('http');
+const fs = require('fs');
+const qs = require('querystring');
+
+let authenticationObject = {
+  something: 'imadethisup',
+};
+
+let existingFiles = ['css', '404.html', 'index.html', '.keep'];
+
+let elementNameReceived = '';
+let elementSymbolReceived = '';
+let elementAtomicNumberReceived = 0;
+let elementDescriptionReceived = '';
+let fileName = '';
+
+const server = http.createServer((req, res) => {
+  let reqURL = '/index.html';
+
+  if (req.url !== '/') {
+    if (fs.existsSync(`./public${req.url}`)) {
+      reqURL = req.url;
+    } else {
+      reqURL = '/404.html';
+    }
+  }
+
+  if (req.method === 'GET') {
+    fs.readFile(`./public${reqURL}`, 'UTF-8', function(err, data) {
```

Comment:
You should add a return to the beginning of this line so that none of the code below the 'get' is executed by accident.